src/hooks/useFetchAllOrgJobs.ts

```diff
@@ -0,0 +1,33 @@
+import { useQuery } from '@tanstack/react-query';
+import config from 'config';
+import { getAccessToken } from 'utils/authFn';
+import { handleApiErrors } from 'utils/handleApiErrors';
+
+const useFetchAllOrgJobs = (organisationId: number) => {
+  const accessToken = getAccessToken();
+
+  const { data, isLoading } = useQuery(
+    ['org_jobs_limit', organisationId],
```

Comment:
There's one more `useQuery` hook which is fetching organisation jobs. Can we eliminate that?
Its query key is `[orgJobs]` and then in that file use the data from here.